pandas/plotting/_core.py

```diff
@@ -2835,18 +2835,37 @@ def area(self, x=None, y=None, **kwds):
 
     def pie(self, y=None, **kwds):
         """
-        Pie chart
+        Generate a pie plot.
+
+        A pie plot is a representation of the distribution of numerical data
+        in a DataFrame column. This function wraps the `matplotlib.pyplot.pie`
+        function for specified column.
 
         Parameters
         ----------
-        y : label or position, optional
-            Column to plot.
-        `**kwds` : optional
+        y : str or int, optional
+            Label or position.
+            If not provided, ``subplots=True`` argument must be passed.
```

Comment:
Really? What happens if we don't pass `subplots=True` and we don't pass a `y` argument? Does it raise an error? What does the `subplots=True` argument change?

Reply:
Yes. This error is raised:
```ValueError: pie requires either y column or 'subplots=True'```
with subplots=True, pie() returns an array of plots for all columns